Sources/DatadogCrashReporting/PLCrashReporterIntegration/CrashReport.swift

```diff
@@ -104,6 +106,9 @@ internal struct BinaryImageInfo {
 
 /// Intermediate representation of `PLCrashReportStackFrameInfo`.
 internal struct StackFrame {
+    /// The number of this frame in the stack trace.
+    /// This must be recorded as less meaningful stack frames might be removed when minifying the `CrashReport`.
+    var number: Int
```

Comment:
how is this being used...the way I see it you remove from the middle of the set not from the end...